tasks/__init__.py

```diff
@@ -790,25 +790,25 @@ def hotfix(name, finish=False, push=False):
     print('Renaming branch...')
 
     new_branch_name = 'hotfix/{}'.format(next_patch_version)
-    run('git checkout {}'.format(name), echo=True)
-    run('git branch -m {}'.format(new_branch_name), echo=True)
+    ctx.run('git checkout {}'.format(name), echo=True)
+    ctx.run('git branch -m {}'.format(new_branch_name), echo=True)
     if finish:
         run('git flow hotfix finish {}'.format(next_patch_version), echo=True, pty=True)
```

Comment:
This should be changed to `ctx.run`. You should also be able to remove the `run` import.